crates/gitbutler-branch-actions/src/stack.rs

```diff
@@ -248,7 +257,10 @@ pub(crate) fn stack_series(
         // Reverse first instead of later, so that we catch the first integrated commit
         for commit in series.clone().local_commits.iter().rev() {
             if !is_integrated {
-                is_integrated = check_commit.is_integrated(commit)?;
+                is_integrated = commits
```

Comment:
This seems to save time and be reasonable in comparison to what it was before. However, since the stack can have its own remote I am not sure if the `check_commit` instance is configured correctly.